CheckerboardKarel.java

```diff
@@ -1,16 +1,38 @@
-/*
- * File: CheckerboardKarel.java
- * ----------------------------
- * When you finish writing it, the CheckerboardKarel class should draw
- * a checkerboard using beepers, as described in Assignment 1.  You
- * should make sure that your program works for all of the sample
- * worlds supplied in the starter folder.
- */
-
-import stanford.karel.*;
-
-public class CheckerboardKarel extends SuperKarel {
-
-	// You fill in this part
-
-}
+/*
+ * File: CheckerboardKarel.java
+ * ----------------------------
+ * When you finish writing it, the CheckerboardKarel class should draw
+ * a checkerboard using beepers, as described in Assignment 1.  You
+ * should make sure that your program works for all of the sample
+ * worlds supplied in the starter folder.
+ */
+
+import stanford.karel.*;
+
+public class CheckerboardKarel extends SuperKarel {
+
+	// You fill in this part
+
+	public void run() {
+		
+		placeFirstBeeper();
+		myMove();
+		
+		
+	}
+
+			
+		private void myMove(){

```

Comment:
This is a strange name for a method: myMove doesn't help the reader know what the method is doing.